### PR TITLE
Adds Crowd Control Specialist profession. (DRAFT)

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary

```SUMMARY: Content "Adds new Crowd Control Specialist profession"```

#### Purpose of change
I wanted to play a character that could play around with the grenade launchers in the early game and i noticed there was no starting professions with access to a grenade launcher and ammo, which seemed sad.

#### Describe the solution
Add Crowd Control Specialist. Description:

When the riots started getting really bad, the brass issued you a new set of gear and told you to put them down hard.  You deserted rather than follow those orders, and now you think the new gear might come in handy after all.

This riot cop comes with a variety of grenades for his 6 round grenade launcher but does not carry a standard gun.

#### Testing
Started a new game with this profession and made sure he had all items.

#### Additional context
My original goal here was to have the cop spawn with his MGL loaded with 6 frag grenades, and then have 4 frag grenades in the left grenade pouch, and 4 buckshot grenades in the right grenade pouch, and perhaps a spare set of each in his inventory.

Instead i entered a 3 hour long struggle with the json file, i could not find a way to start the MGL loaded with the proper grenades, have the leg pouches come with 4 (max) rather than 6 grenades, and to have more than 6 grenades of one variant spawn in the inventory. 

The profession currently spawns with 6 of each 40mm grenade in his inventory to load as he chooses, but if someone knows how to give him the grenades i intended please let me know.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
Ralreegorganon discovered a glitch that is causing my item spawning issues, i'll mark this as a draft until that gets fixed and i can make the grenades spawn properly.

**Comment by carmakazi** [comment]
I like the idea of having a launcher-based class, but from the perspective of nofun realism, the weapon of choice doesn't make a whole lot of sense.  While detailed acquisition information from the myriad of US law enforcement agencies is hard to find, from what I can tell, none of them have purchased the MGL.  That would leave the explanation that, while the world was going to hell, they somehow purchased them or acquired them from Army or Marine stocks, two fairly unlikely scenarios.  40mm grenades are plentiful in military stock, those launchers aren't.

If a police officer were to have a launcher, it would likely be one the department already had on hand for riot control.  From what I can tell, many of these are dated single-shot launchers, and many of them are 37mm, incompatible with explosive grenades.  However, it wouldn't be improbable for a department to have 40mm launchers for that task either.  Even saying the military, in crisis, sent out some dated launchers from old stock like the M79 would be a bit more probable than handing out M32s that they would most likely want to use themselves.  Hell, that even has precedence:
https://www.masslive.com/news/2014/07/aclu_calls_into_question_why_w.html
Those weapons in particular were eventually recalled by the president, but you get my point.

As for straight game reasons, the MGL is possibly one of the most destructive handheld weapons you can get in the game and I don't really agree with giving it as a starting weapon.  It'd be like giving a military sniper the M107 .50 off the bat.  A single-shot launcher with a dozen or so grenades would still give the player a powerful weapon, a better dip into launchers than is currently available, but require them to pay attention to the action economy of reloading and force them to count their limited ammo a little better.  In exchange for the loss in firepower, I'd go ahead and give them a pistol like most/all of the other police specialties.

TL:DR: I'd personally give them a single-shot launcher like the M79 or M320 and a pistol instead of the very powerful M32, but it's your PR and that's just my opinion.

**Comment by Brian-Otten** [comment]
On second review i think it does make sense to swap it to a single shot launcher. I may have gotten a little carried away. Will swap it once i go back and fix the grenade loadout now the bug has been fixed.
